http/execute_request.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import json
+import requests
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--method', dest='method', required=True,
+            choices={'GET', 'POST', 'PUT'})
+    parser.add_argument('--url', dest='url', required=True)
+    parser.add_argument('--authorization-key', dest='authorization_key',
+            required=False)
+    parser.add_argument('--authorization-value', dest='authorization_value',
+            required=False)
```

Comment:
Let's move this to a single key of `authorization-header` and if it's set by the user, then send whatever value without modification under the 'Authorization' header. This way it's more flexible for Bearer and other types of header content.